Cython/Compiler/Code.py

```diff
@@ -1494,6 +1494,23 @@ def generate_string_constants(self):
             w = self.parts['pystring_table']
             w.putln("")
             w.putln("static __Pyx_StringTabEntry %s[] = {" % Naming.stringtab_cname)
+            if py_string.py3str_cstring:
+                w.putln("#if PY_MAJOR_VERSION >= 3")
+                w_py3str_writer = w.insertion_point()
+                w.putln("#else")
+            w.putln("#if CYTHON_COMPILING_IN_LIMITED_API")
```

Comment:
This seems wrong. Only Py3 has the limited API.

Reply:
It is (but it's also wrong in master). For example the test "cython3_no_unicode_literals" crashes with -DCYTHON_LIMITED_API. I'll get this fixed though